spring-boot-starter/spring-boot-starter-discovery/src/main/java/org/apache/servicecomb/springboot/starter/discovery/CseDiscoveryClient.java

```diff
@@ -44,23 +46,62 @@ public String description() {
 
   @Override
   public List<ServiceInstance> getInstances(final String serviceId) {
+
+    class InstanceDiscoveryFilter implements DiscoveryFilter {
+
+      @Override
+      public int getOrder() {
+        return Integer.MAX_VALUE;
```

Comment:
should give a change to filter List<ServiceInstance>
so set order to be max int is not a good idea
we can set it to be: (int) Short.MAX_VALUE;